codec-compression/src/test/java/io/netty/handler/codec/compression/ZstdEncoderTest.java

```diff
@@ -43,6 +43,14 @@ public void setup() {
         when(ctx.alloc()).thenReturn(ByteBufAllocator.DEFAULT);
     }
 
+    public static ByteBuf[] hugeData() {
+        final byte[] bytesHuge = new byte[256 * 1024 * 1024];
```

Comment:
Can we just use something slightly larger then 32MB so we still validate that the fix work while not need to reserve too much memory for the test ?